tests/suites/test_suite_ssl.function

```diff
@@ -5277,3 +5277,14 @@ void conf_group()
     mbedtls_ssl_config_free( &conf );
 }
 /* END_CASE */
+
+/* BEGIN_CASE depends_on:MBEDTLS_TIMING_C:MBEDTLS_HAVE_TIME */
+void timing_accessor( )
```

Comment:
`void timing_accessor( )` -> `void timing_accessor( void )` ?

Reply:
If this was C code, only consumed by C compilers, I'd fully agree. But actually it's mostly C code that's mangled by a python script to produce a result that's going to be passed to the C compiler. I'm not sure* the python script likes `( void )`, and other test functions use `( )`, so I'd keep `( )` here.

*Ok, I checked if we have some documentation, it looks like it's a "use the source, Luke" kind of situation, but [this](https://github.com/ARMmbed/mbedtls/blob/development/tests/scripts/generate_test_code.py#L494) suggests `( void )` won't work.

Reply:
Indeed, for future reference the python script won't let you use void, as it has a list of allowed argument types.